xbmc/interfaces/legacy/Control.cpp

```diff
@@ -150,7 +150,7 @@ namespace XBMCAddon
     {
       if (!pGUIControl) return NULL;
 
-      LOCKGUI;
+      XBMCAddonUtils::GuiLock(languageHook);
```

Comment:
Would be interesting.
Seems like a weird namespace related issue to me.
If you retrieve the languageHook via the getter method `GetLanguageHook` it compiles too.